tests/core/test_processor.py

```diff
@@ -245,11 +243,93 @@
 
     tracker_0 = default_processor.tracker_store.retrieve(sender_ids[0])
     # there should be no utter_greet action
-    assert ActionExecuted("utter_greet") not in tracker_0.events
+    assert (
+        UserUttered(
+            f"{EXTERNAL_MESSAGE_PREFIX}greet",
+            intent={"name": "greet", IS_EXTERNAL: True},
+        )
+        not in tracker_0.events
+    )
 
     tracker_1 = default_processor.tracker_store.retrieve(sender_ids[1])
     # there should be utter_greet action
-    assert ActionExecuted("utter_greet") in tracker_1.events
+    assert (
+        UserUttered(
+            f"{EXTERNAL_MESSAGE_PREFIX}greet",
+            intent={"name": "greet", IS_EXTERNAL: True},
+        )
+        in tracker_1.events
+    )
+
+
+async def test_reminder_cancelled_2(
+    default_channel: CollectingOutputChannel, default_processor: MessageProcessor
+):
+    """Here we create several reminders and cancel them by intent, entities or name."""
```

Comment:
one test should always just test one thing. If you test canceling by different things (intent, entities) then I'd create separate tests (`test_cancel_reminder_by_intent`, `test_cancel_reminder_by_entities`)

Reply:
But things might interfere with each other, right? So I'll leave this test and add the individual tests as well.